src/stylesheets/blocks/_question.scss

```diff
@@ -0,0 +1,67 @@
+.question-section {
+  background-color: color(mint-green);
+  
+  @media (screen(lg)) {
+    margin-top: rem(-18);
+    padding-bottom: rem(170);
+  }
+}
+
+.wrapper-title__question {
+  .lg-text {
+    text-align: left;
+
+    @media (screen(lg)) {
+      text-align: center;
+    }
+  }
+}
+
+.question-section__mobile-phone {
+  background-image: url('../../images/SmartPhone_Question-01.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  float: right;
+  height: rem(220);
+  margin: rem(-65) rem(-75) 0 0;
+  width: rem(150);
+
+  @media (screen(md)) {
+    height: rem(300);
+    width: rem(200);
+  }
+
+  @media (screen(lg)) {
+    background: none;
+    height: 0;
+    width: 0;
```

Comment:
Instead of hiding with these 3 properties and resetting them back for the `--togle-img` version, I'd try doing modifiers for both versions, e.g. `-mobile-version` and `-desktop-version` (or something like that). Then you can just do `display: none` on the mobile one for desktop, and do the same for the desktop one for mobile, and you won't have to unset and re-apply other styles below.